pastas/stressmodels.py

```diff
@@ -1678,6 +1731,29 @@ def simulate(
 
         return h
 
+    def to_dict(self, series: Optional[bool] = True):
```

Comment:
No `Optional[bool]` but `bool`.

`Optional` = `Union[type, None]` = `type | None`

Reply:
Thanks for the type hint. I'll change it. I thought this is that now this argument is optional but I guess not.